uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1,154 +1,103 @@
-<section class="panel panel-default panel-default-plain">
-  <header class="panel-heading">
-    <h2 class="panel-title">Contract Entitlements</h2>
-    <hr/>
-  </header>
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Contract Entitlements</h3>
+  </div>
   <div class="table-responsive">
     <table class="table">
-      <tr>
-        <td>
-          <strong>Position</strong>
-        </td>
-        <td>
-          <strong>Start Date</strong>
-        </td>
-        <td>
-          <strong>End Date</strong>
-        </td>
-        <td>
-          <strong>Sick</strong>
-        </td>
-        <td>
-          <strong>Vacation</strong>
-        </td>
-        <td>
-          <strong>Marernity</strong>
-        </td>
-        <td>
-          <strong>Paternity</strong>
-        </td>
-        <td>
-          <strong>TOIL</strong>
-        </td>
-        <td>
-          <strong>TOIL (Credit)</strong>
-        </td>
-        <td>
-          <strong>Other</strong>
-        </td>
-      </tr>
-      <tr>
-        <td>Developer</td>
-        <td>01/01/2015</td>
-        <td>01/01/2016</td>
-        <td>20</td>
-        <td>30</td>
-        <td>40</td>
-        <td>20</td>
-        <td>30</td>
-        <td>40</td>
-        <td>20</td>
-      </tr>
-      <tr>
-        <td>Manager</td>
-        <td>01/01/2015</td>
-        <td></td>
-        <td>20</td>
-        <td>30</td>
-        <td>40</td>
-        <td>20</td>
-        <td>30</td>
-        <td>40</td>
-        <td>20</td>
-      </tr>
+      <thead>
+        <tr>
+          <th style="width:140px">Position</th>
```

Comment:
What's the need to use inline styles here? should we not use css class or something for the width of the column heading?